contribs/freight/src/main/java/org/matsim/contrib/freight/utils/FreightUtils.java

```diff
@@ -209,6 +209,20 @@ public static CarrierVehicleTypes getCarrierVehicleTypes(Scenario scenario) {
 		}
 		return types;
 	}
+	
+	/** Adds the CarrierVehicleTypes to the scenario.
+	 * @param scenario
+	 * @param newTypes
+	 */
+	public static void addCarrierVehicleTypes(Scenario scenario, CarrierVehicleTypes newTypes) {
```

Comment:
I would prefer to have it more analog to the `addOrGetCarriers` method in line 182ff. --> not set the values directly; This seems to be closer to the MATSim standard way of doing it.

The types can be set then outside the FreightUtils